database/insertData/_helpers/core_mutations.js

```diff
@@ -254,6 +255,74 @@ exports.coreActions = `
         isReview: false #Required since we're updating an application status
       }
     }
+    #
+    # Change outcome of application to APPROVED if last decision is CONFORM
+    #
+    {
+      actionCode: "changeOutcome"
+      trigger: ON_REVIEW_SUBMIT
+      sequence: 8
+      condition: {
+        operator: "AND"
+        children: [
+          {
+            operator: "="
+            children: [
+              {
+                operator: "objectProperties"
+                children: [
+                  "applicationData.reviewData.latestDecision.decision"
+                ]
+              }
+              "CONFORM"
+            ]
+          }
+          {
+            operator: "objectProperties"
+            children: ["applicationData.reviewData.isLastLevel"]
+          }
+          {
+            operator: "objectProperties"
+            children: ["applicationData.reviewData.isLastStage"]
+          }
+        ]
+      }
+      parameterQueries: { newOutcome: { value: "APPROVED" } }
```

Comment:
can we make changeOutcome actions like this as one ?

I used template builder (to also show what this sort of config would look like):
https://www.dropbox.com/s/ri0xrj7mu5jcjhz/changeOutcome-TP.mov?dl=0

condition:
```JSON
 "operator": "AND",
 "children": [
  {
   "operator": "!=",
   "children": [
    {
     "operator": "objectProperties",
     "children": [
      "applicationData.reviewData.latestDecision.decision"
     ]
    },
    "LIST_OF_QUESTIONS"
   ]
  },
  {
   "operator": "objectProperties",
   "children": [
    "applicationData.reviewData.isLastLevel"
   ]
  },
  {
   "operator": "objectProperties",
   "children": [
    "applicationData.reviewData.isLastStage",
    null
   ]
  }
 ]
}
```
parameterQueries
```JSON
{
 "newOutcome": {
  "operator": "?",
  "children": [
   {
    "operator": "=",
    "children": [
     {
      "operator": "objectProperties",
      "children": [
       "applicationData.reviewData.latestDecision.decision",
       null
      ]
     },
     "CONFORM"
    ]
   },
   "APPROVED",
   "REJECTED"
  ]
 }
}
```

Reply:
Nice! I was still a little lost on the config, the layout is hard to understand where to look for things. But it's nice this is now registered so I can go back to this when neeeded